web/templates/popovers/user_card/user_card_popover.hbs

```diff
@@ -1,176 +1,207 @@
                     {{/if}}
                 </a>
             </li>
-            <li>
-                <a href="{{ sent_by_url }}" class="narrow_to_messages_sent">
-                    <i class="fa fa-paper-plane" aria-hidden="true"></i> {{t "View messages sent" }}
-                </a>
+        {{/if}}
+        {{#unless is_me}}
+            <li role="none" class="popover-menu-list-item link-item hidden-for-spectators">
+                {{#if has_message_context}}
+                    <a role="menuitem" class="popover-menu-link mention_user" tabindex="0">
+                        <i class="popover-menu-icon zulip-icon zulip-icon-at-sign" aria-hidden="true"></i>
+                        {{#if is_bot}}
+                            <span class="popover-menu-label">{{t "Reply mentioning bot" }}</span>
+                        {{else}}
+                            <span class="popover-menu-label">{{t "Reply mentioning user" }}</span>
+                        {{/if}}
+                        {{#if is_sender_popover}}
+                            {{popover_hotkey_hints "@"}}
+                        {{/if}}
+                    </a>
+                {{else}}
+                    <a role="menuitem" class="popover-menu-link copy_mention_syntax" tabindex="0" data-clipboard-text="{{ user_mention_syntax }}">
+                        <i class="popover-menu-icon fa zulip-icon zulip-icon-at-sign" aria-hidden="true"></i>
+                        <span class="popover-menu-label">{{t "Copy mention syntax" }}</span>
+                        {{#if is_sender_popover}}
+                            {{popover_hotkey_hints "@"}}
+                        {{/if}}
+                    </a>
+                {{/if}}
             </li>
         {{/unless}}
+        {{#if is_me}}
+            <li role="none" class="popover-menu-list-item link-item hidden-for-spectators">
+                <a role="menuitem" href="/#settings/profile" class="popover-menu-link" tabindex="0">
+                    <i class="popover-menu-icon zulip-icon zulip-icon-tool" aria-hidden="true"></i>
+                    <span class="popover-menu-label">{{t "Edit your profile" }}</span>
+                </a>
+            </li>
+        {{/if}}
+        <li role="separator" class="popover-menu-separator hidden-for-spectators"></li>
+        <li role="none" class="popover-menu-list-item link-item">
+            <a role="menuitem" href="{{ pm_with_url }}" class="narrow_to_private_messages popover-menu-link hidden-for-spectators" tabindex="0">
+                <i class="popover-menu-icon zulip-icon zulip-icon-mail" aria-hidden="true"></i>
+                {{#if is_me}}
+                    <span class="popover-menu-label">{{t "View messages with yourself" }}</span>
+                {{else}}
+                    <span class="popover-menu-label">{{t "View direct messages" }}</span>
+                {{/if}}
+            </a>
+        </li>
+        <li role="none" class="popover-menu-list-item link-item">
```

Comment:
This should certainly be a follow-up, possibly after 9.0, but `link-item` should probably be `popover-link-item`.